src/qcodes/parameters/parameter.py

```diff
@@ -261,13 +263,13 @@ def _set_manual_parameter(
         # in the scope of this class.
         # (previous call to `super().__init__` wraps existing
         # get_raw/set_raw into get/set methods)
-        if self.gettable and get_cmd not in (None, False):
+        if self._gettable and get_cmd not in (None, False):
```

Comment:
Lets expose a property on ParameterBase (_has_get_raw,_has_set_raw) that we can check here. That will make the code more readable and expose what we are actually checking